### PR TITLE
Update the Changelog/Credits

### PR DESCRIPTION
**Pull request by Arrow768**
Update the Changelog/Credits to reflect the aurora team

### CODE REVIEW

**Comment by NanakoAC** [comment]
audio is kind of a thing that i do, ive adde seven audio files across two PRs so far, expecting to do more in future. I'm moderately proficient in cutting and editing with audacity

File: html/changelog.html [hunk 1]
```diff
@@ -24,36 +24,36 @@
...
-			<font size='2'><b>Sprites:</b> Apple_Master, Arcalane, Chinsky, CompactNinja, Deus Dactyl, Erthilo, Flashkirby, JoeyJo0, Miniature, Searif, Xenone, faux<br></font>
-			<font size='2'><b>Sounds:</b> Aryn<br></font>
-            <font size='2'><b>Main Testers:</b> Anyone who has submitted a bug to the issue tracker<br></font>
-			<font size='2'><b>Thanks to:</b> /tg/ station, /vg/station, GoonStation devs, the original SpaceStation developers and Invisty for the title image.<br> Also a thanks to anybody who has contributed who is not listed here :( Ask to be added here on irc.</font>
-			<font size='2' color='red'><b><br>Have a bug to report?</b> Visit our <a href="https://github.com/Baystation12/Baystation12/issues?labels=Bug&state=open">Issue Tracker</a>.<br></font>
+			<font size='2'><b>Current development team :</b> <a href='http://forums.aurorastation.org/memberlist.php?mode=group&g=9'>-Click Here-</a><br></font>
+			<font size='2'><b>Code:</b> Soundscopes, Skull132, LadyofRavens, Ryan, LordLag, LordFowl, Arrow768, Alberyk, Nanako<br></font>
+      <font size='2'><b>Map:</b> LordFowl, Keknar, PumpkinSlice<br></font>
```
Comment (skull132) [inline html/changelog.html]: Wittme91 to mappers.

File: html/changelog.html [hunk 2]
```diff
@@ -1,20 +1,20 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
 <html>
 <head>
-  <title>Baystation 12 Changelog</title>
+  <title>Aurorastation 12 Changelog</title>
```
Comment (skull132) [inline html/changelog.html]: This should probably be `Aurorastation Changelog`, so nix the 12.

File: html/changelog.html [hunk 3]
```diff
@@ -24,36 +24,36 @@
...
-			<font size='2' color='red'><b><br>Have a bug to report?</b> Visit our <a href="https://github.com/Baystation12/Baystation12/issues?labels=Bug&state=open">Issue Tracker</a>.<br></font>
+			<font size='2'><b>Current development team :</b> <a href='http://forums.aurorastation.org/memberlist.php?mode=group&g=9'>-Click Here-</a><br></font>
+			<font size='2'><b>Code:</b> Soundscopes, Skull132, LadyofRavens, Ryan, LordLag, LordFowl, Arrow768, Alberyk, Nanako<br></font>
+      <font size='2'><b>Map:</b> LordFowl, Keknar, PumpkinSlice<br></font>
+			<font size='2'><b>Sprites:</b> Killerhurtz, CakeIsOssim, FireandGlory, FFrances, Rechkalov<br></font>
+			<font size='2'><b>Sounds:</b> Killerhurtz<br></font>
+      <font size='2'><b>Main Testers:</b> Anyone who has submitted a bug to the issue tracker<br></font>
+			<font size='2'><b>Thanks to:</b> BayStation, /tg/ station, /vg/station, GoonStation devs, the original SpaceStation developers and Invisty for the title image.<br> Also a thanks to anybody who has contributed who is not listed here :( Ask to be added here on the forums</font>
```
Comment (skull132) [inline html/changelog.html]: Baystation12, as opposed to Baystation.

File: html/changelog.html (hunk 1 above)
Comment (LordFowl) [inline html/changelog.html]: Juani2400 should also be added to mappers.

**Comment by Arrow768** [comment]
Related to #589

**Pull request merged**